content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -76,9 +76,15 @@ The following configuration options behave consistently across the latest versio
 
 ### Traces
 
+`DD_APM_TRACING_ENABLED`
+: **Default**: `true` <br>
+**Supported Input**: Boolean <br>
+**Description**: Enables or disables sending traces from the application, without impacting other library features such as profiling, AAP, DSM, and more.
```

Comment:
```suggestion
**Description**: Enables or disables sending traces from the application, without impacting other library features such as profiling, Datadog App and API Protection (AAP), Data Streams Monitoring (DSM), and more.
```
I realize these were not spelled out before but since AAP is a fairly new acronym we should spell it out here, as well as DSM.